cfme/intelligence/chargeback.py

```diff
@@ -3,88 +3,113 @@
 from copy import copy
 from functools import partial
 
-from selenium.webdriver.common.by import By
+from cached_property import cached_property
+
 from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic.utils import ParametrizedLocator, ParametrizedString
+from widgetastic.widget import Text, ParametrizedView
+from widgetastic_manageiq import Select
+from widgetastic_patternfly import Input, Button, Dropdown
 
 from cfme import BaseLoggedInPage
-import cfme.web_ui.accordion as accordion
-import cfme.web_ui.toolbar as tb
 import cfme.fixtures.pytest_selenium as sel
-from cfme.web_ui import Form, Select, Tree, fill, flash, form_buttons, match_location
-from cfme.web_ui.form_buttons import FormButton
+import cfme.web_ui.accordion as accordion
+from cfme.web_ui import Form, Tree, fill, flash, form_buttons, match_location
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
 from utils.pretty import Pretty
 from utils.update import Updateable
 from utils.version import LOWEST, pick
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
+
 
 assignment_tree = Tree("//div[@id='cb_assignments_treebox']/ul")
-tb_select = partial(tb.select, "Configuration")
+
 
 match_page = partial(match_location, controller='chargeback',
                      title='Chargeback')
 
 
 class ChargebackView(BaseLoggedInPage):
     @property
-    def is_displayed(self):
+    def in_explorer(self):
         return (
             self.logged_in_as_current_user and
             self.navigation.currently_selected == ['Cloud Intel', 'Chargeback'])
 
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and
+            self.title.text == 'Compute Chargeback Rates')
```

Comment:
Ensure this condition is unique for this particular view. Applies to all such conditions. And the property should be named in_chargeback instead of in_explorer.